### ISSUE TITLE
Cannot run qwen2 7B, 1.5b

### ISSUE THREAD

**Issue by SAXN-SYNX** [ollama/ollama#4907]
### Shows error while running it.

```
llama_model_loader: Dumping metadata keys/values. Note: KV overrides do not apply in this output.
llama_model_loader: - kv   0:                       general.architecture str              = qwen2
llama_model_loader: - kv   1:                               general.name str              = Qwen2-7B-Instruct
llama_model_loader: - kv   2:                          qwen2.block_count u32              = 28
llama_model_loader: - kv   3:                       qwen2.context_length u32              = 32768
llama_model_loader: - kv   4:                     qwen2.embedding_length u32              = 3584
llama_model_loader: - kv   5:                  qwen2.feed_forward_length u32              = 18944
llama_model_loader: - kv   6:                 qwen2.attention.head_count u32              = 28
llama_model_loader: - kv   7:              qwen2.attention.head_count_kv u32              = 4
llama_model_loader: - kv   8:                       qwen2.rope.freq_base f32              = 1000000.000000
llama_model_loader: - kv   9:     qwen2.attention.layer_norm_rms_epsilon f32              = 0.000001
llama_model_loader: - kv  10:                          general.file_type u32              = 2
llama_model_loader: - kv  11:                       tokenizer.ggml.model str              = gpt2
llama_model_loader: - kv  12:                         tokenizer.ggml.pre str              = qwen2
llama_model_loader: - kv  13:                      tokenizer.ggml.tokens arr[str,152064]  = ["!", "\"", "#", "$", "%", "&", "'", ...
llama_model_loader: - kv  14:                  tokenizer.ggml.token_type arr[i32,152064]  = [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, ...
llama_model_loader: - kv  15:                      tokenizer.ggml.merges arr[str,151387]  = ["Ġ Ġ", "ĠĠ ĠĠ", "i n", "Ġ t",...
llama_model_loader: - kv  16:                tokenizer.ggml.eos_token_id u32              = 151645
llama_model_loader: - kv  17:            tokenizer.ggml.padding_token_id u32              = 151643
llama_model_loader: - kv  18:                tokenizer.ggml.bos_token_id u32              = 151643
llama_model_loader: - kv  19:                    tokenizer.chat_template str              = {% for message in messages %}{% if lo...
llama_model_loader: - kv  20:               general.quantization_version u32              = 2
llama_model_loader: - type  f32:  141 tensors
llama_model_loader: - type q4_0:  197 tensors
llama_model_loader: - type q6_K:    1 tensors
llama_model_load: error loading model: error loading model vocabulary: unknown pre-tokenizer type: 'qwen2'
llama_load_model_from_file: exception loading model
terminate called after throwing an instance of 'std::runtime_error'
  what():  error loading model vocabulary: unknown pre-tokenizer type: 'qwen2'

```

### OS

Linux

### GPU

Nvidia

### CPU

Intel

### Ollama version

0.1.34

**Comment by include** [comment]
+1.

**Comment by boxabirds** [comment]
Lots of people experiencing this in the Discord.

**Comment by schizoposter** [comment]
> Ollama version
> 0.1.34
Try upgrading first, fixed the issue for me.

**Comment by boxabirds** [comment]
I get garbage output with 41. ![image](https://github.com/ollama/ollama/assets/147305/523ea64c-16cb-4670-a590-63871204b723)

**Comment by iplayfast** [comment]
I tried running deepseek-v2 (which it pulled and installed) and after that qwen2 started behaving. (could be a clue?)

**Comment by guinanet** [comment]
Escreva um código em python para somar os números de 1 até 100.

qwen2:1.5b-instruct 3:12 PM
GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG

![image](https://github.com/ollama/ollama/assets/144032090/7d02fc9d-c100-4dd8-8c13-621f6dba271e)

**Comment by jmorganca** [comment]
Hi all, sorry about this. It is fixed in 0.1.42: https://github.com/ollama/ollama/releases/tag/v0.1.42

**Comment by liuaifu** [comment]
@jmorganca 
```
Microsoft Windows [版本 10.0.22631.3593]
(c) Microsoft Corporation。保留所有权利。

C:\Users\laf163>ollama -v
ollama version is 0.1.42

C:\Users\laf163>ollama run qwen2:7b
>>> Why is the sky blue?
#&%$#7456893210

>>> Send a message (/? for help)
```
